### PR TITLE
Redesign display of versions data

### PR DESCRIPTION
**Pull request by beausmith**
## Insert Card Screen

image

## Voter Flow

image

## Poll Worker Screen

image

## Admin Screens

image

image

### CODE REVIEW

File: apps/bmd/src/pages/AdminScreen.tsx [hunk 1]
```diff
@@ -22,11 +26,14 @@ import {
 } from '../utils/date'
 import InputGroup from '../components/InputGroup'
 import Modal from '../components/Modal'
+import VersionsData from '../components/VersionsData'
 
 type Meridian = 'AM' | 'PM'
 
 interface Props {
+  // TODO: replace both of these with machineConfig to get machineId too
```
Comment (eventualbuddha) [inline apps/bmd/src/pages/AdminScreen.tsx:34]: This is from my WIP. Seems like you did include `machineConfig` and can remove this?

File: apps/bmd/src/pages/AdminScreen.tsx [hunk 2]
```diff
@@ -50,6 +59,7 @@ const AdminScreen: React.FC<Props> = ({
   updateAppPrecinctId,
   toggleLiveMode,
   unconfigure,
+  machineConfig,
```
Comment (eventualbuddha) [inline apps/bmd/src/pages/AdminScreen.tsx:62]: If we have this we don't need `appMode` and `codeVersion` separately, right?

**Review by eventualbuddha — APPROVED** [review]

File: apps/bmd/src/AppRoot.tsx [hunk 3]
```diff
@@ -1137,6 +1137,7 @@ const AppRoot: React.FC<Props> = ({
     return (
       <AdminScreen
         appMode={appMode}
+        codeVersion={codeVersion}
```
Comment (carolinemodic) [inline apps/bmd/src/AppRoot.tsx:1140]: +1 to brians comment below, this seems redundant with machineConfig

**Review by carolinemodic — APPROVED** [review]
I agree with brians comments, but otherwise this look good, love the redesign!

**Comment by beausmith** [comment]
@eventualbuddha @carolinemodic - I appended a commit to resolve the issues you noted. Thank you for your reviews. I'm merging…

**Pull request merged**